src/figuresClasses.ts

```diff
@@ -1,11 +1,77 @@
-export interface Figure {}
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
```

Comment:
[CODE STYLE] - The `getArea` function should not accept any parameters according to the checklist. In this case, the interface correctly defines `getArea` without parameters, so make sure the implementations in the classes below also follow this pattern.